### PR TITLE
1045 ability to add line height token in rem units

### PR DESCRIPTION
**Pull request by MightyDevKnight**
https://user-images.githubusercontent.com/102773603/218662952-0fa4609d-ce1d-4f01-a90b-d901d01eaa3c.mp4

### CODE REVIEW

File: src/plugin/figmaTransforms/lineHeight.ts [hunk 1]
```diff
@@ -14,6 +14,11 @@ export function convertLineHeightToFigma(inputValue: string, baseFontSize: strin
       unit: 'PERCENT',
       value: Number(value.slice(0, -1)),
     };
+  } else if (value.match(numbers)) {
+    lineHeight = {
+      unit: 'PERCENT',
```
Comment (six7) [inline src/plugin/figmaTransforms/lineHeight.ts:19]: If the value is a number it should be treated like a px value like currently

**Review by six7 — CHANGES_REQUESTED** [review]

File: src/plugin/figmaTransforms/lineHeight.ts (hunk 1 above)
Comment (MightyDevKnight) [reply src/plugin/figmaTransforms/lineHeight.ts:19]: @six7 Please review this pic
image

Comment (six7) [reply src/plugin/figmaTransforms/lineHeight.ts:19]: yes, lets not do this now. changing it means all tokens of users become changed.

Comment (MightyDevKnight) [reply src/plugin/figmaTransforms/lineHeight.ts:19]: okay

**Review by six7 — APPROVED** [review]

**Pull request merged**